### ISSUE TITLE
How to re-train PoseNet models

### ISSUE THREAD

**Issue by take-kuma** [tensorflow/tfjs#1581]
To get help from the community, we encourage using Stack Overflow and the [`tensorflow.js`](https://stackoverflow.com/questions/tagged/tensorflow.js) tag.

#### TensorFlow.js version
1.1.0

#### Browser version
Chrome 74.0.3729.131(Official Build) on Windows 10

#### Describe the problem or feature request
I want to improve PoseNet's accuracy for my application.
Is it easy to add my own dataset and re-train the PoseNet models?
Are scripts for training available?

I have read the following paper but I didn't understand how to train completely.
[PersonLab: Person Pose Estimation and Instance Segmentation with a Bottom-Up, Part-Based, Geometric Embedding Model](https://arxiv.org/abs/1803.08225)

**Comment by tafsiri** [comment]
Hi @take-kuma, there isn't a direct path we provide to re-training PoseNet. As far as I know the original model and training scripts were not open sourced. You could try looking online to see if other people have implemented the model from the paper (I do see a few repos when i search for PersonLab python—but I can't vouch for any of them).

**Comment by take-kuma** [comment]
@tafsiri, thank you for your advice.
I understand that the official training script is not available to me.
So I'll take a look at other codes as follows:

- https://github.com/octiapp/KerasPersonLab
- https://github.com/scnuhealthy/Tensorflow_PersonLab
- https://github.com/Naykira/personlab

I'm OK to close this thread.

**Comment by kavikode** [comment]
How can we use the pre-trained PoseNet model to apply transfer learning to the addition of custom set of images because I want to include people using wheelchairs and people using walkers and crutches as well?
